migrations/scripts/schema.yml

```diff
@@ -0,0 +1,470 @@
+## Configuration options for the base intermediate schema generator
+##
+## After modifying this file, regenerate the base intermediate schema
+## by running the `generate_schema` script
+
+output_directory: ../common/intermediate_db_schema
+output_filename: 000_base_schema.sql
```

Comment:
Should we merge those 2 options into one?

Reply:
Yes! I explained the original plan at https://github.com/discourse/discourse/pull/24484#discussion_r1457357517